docs/devel/README.md

```diff
@@ -46,7 +48,7 @@ Make necessary changes using your preferred editor (e.g., vi/emacs).
 **4.** Build the RPM
 
 ```bash
-VERSION=1.0 make rpm
+make clean && VERSION=1.0 make rpm
```

Comment:
Actually, looking at the makefile target, `rpm` depends on `clean`. Why did you decide to add the `rm -rf` before? It should've been cleaned either way before `rpmbuild`.

Reply:
it was not cleaning the old file. I think it's save to keep it in the docs.